jina/types/request/data.py

```diff
@@ -98,6 +98,8 @@ def __init__(
         request: Optional[RequestSourceType] = None,
     ):
         self.buffer = None
+        self._buffer_when_load_wo_data = None
```

Comment:
I think this naming is weird. I would like to see. buffer_with_docs, buffer_wo_docs